scripts/setup-ubuntu.sh

```diff
@@ -130,7 +131,8 @@ function install_velox_deps_from_apt {
     bison \
     flex \
     libfl-dev \
-    tzdata
+    tzdata \
+    pkg-config
```

Comment:
This is already added on line 95 and part of the build config. It is not a direct build dependency.

Reply:
Removed pkg-config. It was not present in some older versions of Velox, which is why I had added it here.